Source/SwiftLintFramework/Extensions/SwiftExpressionCallKind.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SwiftExpressionCallKind.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 11/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum SwiftExpressionCallKind: String {
```

Comment:
I think that is fine by now, but wouldn't it be weird once this enum is on `SourceKittenFramework`?